README.md

```diff
@@ -81,165 +89,198 @@ function couponCounter(totalAmount, location) {
-`1`.
+Here, because we didn't pass a start value, `reduce()` sets the accumulator
+variable equal to the first element in the array and doesn't pass that value
+into the reducer. As a result, the first element (`1`) _does not get doubled_
+and the final total is `11` rather than `12`.
 
-Let's look at a more complex example now. Say we want to create a roster of
-student artists based on their discipline of art for their final showcase.
-Our start value might look like this:
+To be safe, it is best to always pass a start value when calling `reduce()`. Of
+course, that initial value can be anything we like:
 
 ```js
-const artsShowcases = {
-  Dance: [],
-  Visual: [],
-  Music: [],
-  Theater: [],
-  Writing: [],
-};
-```
-
-Imagine we also have a `studentSorter` object that includes a `showcaseAssign()`
-method. That method takes the name of a student as its argument and returns the
-name of the showcase the student should be assigned to. Note that we have not
-coded out the `showcaseAssign()` method — the details of how it would work are
-not important for our purposes. What's important to remember is that the method
-takes the name of a student as an argument and returns one of the five showcases:
-"Dance", "Visual", "Music", "Theater" or "Writing". We want to call the method
-for each element in our input array (each student's name), get the value of the
-showcase that's returned, and add the student's name to the array for that
-showcase in the `artsShowcases` object.
-
-To do that, we will call reduce on our input array, `incomingStudents`, which
-contains the names of all incoming students, passing a callback function and the
-start value of `artsShowcases` as the arguments. The callback is where we'll
-push each student name into the appropriate showcase:
-
-```js
-incomingStudents.reduce(function (showcases, student) {
-  showcases[studentSorter.showcaseAssign(student)].push(student);
-}, artsShowcases);
+const doubledAndSummedFromTen = nums.reduce(doubleAndSum, 10);
+// => 22
 ```
 
-Let's break this down: `.reduce()` executes the callback for each student name
-in turn. Inside the callback, the `studentSorter.showcaseAssign()` method is
-called with the current student name as its argument. `showcaseAssign()` returns
-the name of an Arts Showcase, which is then used as the key to access the
-correct array in the `artsShowcases` object and push the student's name into it.
-The iteration then continues to the next element in the array, passing the next
-student name and the updated value of `artsShowcases` as the arguments. Once
-`reduce()` has iterated through all the students in `incomingStudents`, it
-returns the final value of `artsShowcases`.
+**REMOVED EXAMPLE WITH OBJECT AS RETURN VALUE - NEEDS TO MOVE LATER**
 
-We can then access the list of students in any Arts Showcase:
-
-```js
-artsShowcases["Visual"]; //=> [yishayGarbasz, wuTsang, mickaleneThomas]
-```
+**TAKE OUT LAB PORTION?**
```

Comment:
Personally I would agree with this, but I generally think labs/lessons should be separated. Maybe we should discuss this in a more general sense?

Reply:
Agree it may be easier on students to review lesson content and have labs separately that way they can take a break and not get overwhelmed with reading and immediately doing a lab. This is also a good way of "chunking" content in the program to avoid cognitive overload on students